pkg/reconciler/intevents/pullsubscription/keda/pullsubscription.go

```diff
@@ -81,6 +81,7 @@ func (r *Reconciler) ReconcileScaledObject(ctx context.Context, ra *appsv1.Deplo
 		existing.Spec = ra.Spec
 		existing, err = r.KubeClientSet.AppsV1().Deployments(src.Namespace).Update(existing)
 		if err != nil {
+			src.Status.MarkDeployedFailed("Error updating the Receive Adapter", err.Error())
```

Comment:
Shouldn't this be 

```
src.Status.MarkDeployedFailed("ReceiveAdapterUpdateFailed", "Error updating the Receive Adapter: %s", err.Error())
```
?

As is, it is missing the reason and the format string doesn't include any format directives.

Reply:
Thanks for catching this. Fixed with this PR https://github.com/google/knative-gcp/pull/1721